app/src/main/res/layout/item_tabs.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout>
```

Comment:
should remove layout tag as you don't need to use data tag

Reply:
i used it to use databinding for type and null safety

Reply:
👍